ztp/tools/get-kustomize-version/get-gitops-operator-kustomize-version.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+
+# Script to determine the kustomize version used in the corresponding argocd container
+# of the current openshift-gitops operator.
+
+# Check for empty parameters.
+if [[ -z $1 || -z $2 || -z $3 || -z $4 ]]; then
+  exit 1
+fi
+
+# From now on return 0 even in case of error since we default to a kustomize version.
+
+# Save the parameters to variables with meaningful names.
+# Usually the OCP version.
+index_tag=$1
+# ztp/gitops-subscriptions/argocd/deployment/openshift-gitops-operator.yaml
+openshift_gitops_operator_sub=$2
+# Default kustomize version.
+default_kustomize_version=$3
+# The name of the local test directory.
+test_dir=$4
+
+# Used files.
+index_results_file="$test_dir/index_results"
+kustomize_version_file="$test_dir/kustomize_version"
+
+# Write the default kustomize version to file.
+mkdir -p $test_dir
+echo "$default_kustomize_version" > "$kustomize_version_file"
+
+cleanup() {
+    rm -f $test_dir
+}
+
+# Check the second parameter points to a file that exists.
+if ! test -f $2; then
+  exit 0
+fi
+
+
+# Render file based catalog from redhat-operator-index.
+podman run --rm registry.redhat.io/redhat/redhat-operator-index:$index_tag render /configs/ > $index_results_file
+if [[ $? -ne 0 ]]; then
+    exit 0
+fi
+
+# Extract the latest desired openshift-gitops channel.
+gitops_channel=$(cat $openshift_gitops_operator_sub | grep "channel: gitops-" | awk '{print $2}')
```

Comment:
awk can also grep; you could shorten this
```suggestion
gitops_channel=$(cat $openshift_gitops_operator_sub | awk '/channel: gitops-/ {print $2}')
```